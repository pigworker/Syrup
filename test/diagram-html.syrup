not(<Bit>) -> <Bit>
not(x) = nand(x, x)

display not