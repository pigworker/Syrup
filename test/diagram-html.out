<style>
  .syrup-code {
    display: block;
    font-family: monospace;
    font-size: 17px;
    white-space: pre;
    margin: 1em 0;
  }
  .syrup-happy:before {
    content: "\2705";
    padding: 0 6px 0 0;
  }
  .syrup-comment:before {
    content: "\2705";
    padding: 0 6px 0 0;
  }
  .syrup-warning:before {
    content: "\26A0\FE0F";
    padding: 0 6px 0 0;
  }
  .syrup-error:before {
    content: "\274C";
    padding: 0 6px 0 0;
  }
  .syrup-internal:before {
    content: "\1F480";
    padding: 0 6px 0 0;
  }

</style>
<div class="syrup-comment">Circuit <code><span class="syrup-function">not</span></code> is defined.</div><br>
<br>
<div class="syrup-happy">not implemented yet</div>